src/components/SideBar.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { FaTimes } from "react-icons/fa";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+const Sidebar = (props) => {
+  return (
+    <div className={props.sidebar}>
+      <FaTimes
+        className="close"
+        onClick={props.setSidebar}
+      />
+      <ul className="sidebar-list-container">
+        <li className="sidebar-list-container-list"><Link to="/">HOME</Link></li>
+        <li className="sidebar-list-container-list"><Link to="/about">ABOUT</Link></li>
+      </ul>
+    </div>
+  );
+};
+
+
+export default Sidebar;
+Sidebar.propTypes = {
```

Comment:
Following clean code practices, Kindly leave a single line space between component exports and component propTypes declarations.